changelogs/fragments/powershell_basic_util.yaml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+- Added Ansible.Basic C# util that contains a module wrapper and handles common functions like argument parsing and module return
```

Comment:
How does adding C# module utils benefit the developer (or the user)? Is it faster? Easier to extend? Cleaner? More familiar? Also, are the new C# module utils replacing the old ones, or just augmenting them?

Reply:
Mostly that we already have PowerShell module_utils that embed C# code inside them. This makes things cleaner, more consistent in how they are loaded, and opens up the possibility of module utils if we do decide to add support for C# modules.